src/jit/compiler.h

```diff
@@ -3096,8 +3100,13 @@ class Compiler
                                 CORINFO_METHOD_HANDLE method,
                                 CORINFO_SIG_INFO*     sig,
                                 bool                  mustExpand);
+    bool impIsTableDrivenHWIntrinsic(HWIntrinsicCategory category);
```

Comment:
nit: A number of these can still be static since they only access data from the table